next.config.js

```diff
@@ -1,5 +1,25 @@
+const { withSentryConfig } = require('@sentry/nextjs')
+const { PHASE_DEVELOPMENT_SERVER, PHASE_PRODUCTION_BUILD } = require('next/constants')
+
 /** @type {import('next').NextConfig} */
 
+const os = require('os')
+
+function getLocalIpAddress() {
+    const interfaces = os.networkInterfaces()
+    for (const name of Object.keys(interfaces)) {
+        for (const iface of interfaces[name]) {
+            if (iface.family === 'IPv4' && !iface.internal) {
+                return iface.address
+            }
+        }
+    }
+    return null
+}
```

Comment:
Used this to log local network url directly when launching the app & test pwa install procedures on mobile. Can remove.

Reply:
I suggest we comment this code out then and add a comment on why it was there - in case we ever need it again for debugging